src/main/java/com/exactpro/th2/common/schema/factory/CommonFactory.java

```diff
@@ -218,6 +222,18 @@ protected Path getOldPathToDictionariesDir() {
     protected ConfigurationManager getConfigurationManager() {
         return configurationManager;
     }
+    
+    public EventBuilder getEventBuilder() {
```

Comment:
Please add two new interfaces: MessageFactory and EventFactory. Instances of these factories can be taken from the AbstractCommonFactory
MessageFactory must have methods: createRawMessage, createParsedMessage
EventFactory must have the same set of methods as static methods in the Event class